application/holder/src/env/os/platform.ts

```diff
@@ -15,6 +16,23 @@ export enum Platform {
     undefined = 'undefined',
 }
 
+let cachedCpuBrandString: string | null = null;
+
+function safeExecSync(command: string, timeout: number): string {
+    try {
+        return execSync(command, { timeout }).toString().trim();
+    } catch (error) {
+        return '';
+    }
```

Comment:
Some logs are needed here.
```
import { scope } from 'platform/env/scope';
import { error } from 'platform/log/utils';
...
try {
        return execSync(command, { timeout }).toString().trim();
    } catch (err) {
        scope.getLogger('PlatformChecker').warn(`Fail to detect arch for darwin. Command "..." gives error: ${error(err)}`);
        return '';
    }
...
```

Reply:
also makes sense to make
```
execSync(command, { timeout }).toString().trim().toLowerCase();
                                                                              ^^^^^^^^^^^^^^
```
It will allow accept also different cases (Intel, intel, INTEL)